pandas/core/arrays/arrow/array.py

```diff
@@ -159,8 +159,25 @@ def to_pyarrow_type(
 
 class ArrowExtensionArray(OpsMixin, ExtensionArray):
     """
-    Base class for ExtensionArray backed by Arrow ChunkedArray.
-    """
+    Pandas ExtensionArray backed by a PyArrow ChunkedArray.
+
+    Parameters
+    ----------
+    values: pyarrow.Array or pyarrow.ChunkedArray
```

Comment:
```suggestion
    values : pyarrow.Array or pyarrow.ChunkedArray
```
I think this space is needed for Sphinx to render this correctly